main.ts

```diff
@@ -543,18 +587,16 @@ export default class VimrcPlugin extends Plugin {
 		}
 	}
 
-	defineFixedLayout(cm: CodeMirror.Editor) {
-		cm.on('keydown', (instance: CodeMirror.Editor, ev: KeyboardEvent) => {
-			if (this.settings.fixedNormalModeLayout) {
-				const keyMap = this.settings.capturedKeyboardMap;
-				if (!this.isInsertMode && !ev.shiftKey &&
-					ev.code in keyMap && ev.key != keyMap[ev.code]) {
-					this.codeMirrorVimObject.handleKey(instance, keyMap[ev.code], 'mapping');
-					ev.preventDefault();
-					return false;
-				}
+	onKeydown = (instance: CodeMirror.Editor, ev: KeyboardEvent) => {

```

Comment:
I may be missing something but this doesn't seem to work for me. A log put before the `if` never seems to be printed. Any clue why?

Reply:
It seems like the way of subscribe CodeMirror event diffs, I try to fix it to work, but I don't know how to debugger in Normal Mode.